akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -253,6 +255,7 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
   private val PoisonPillManifest = "P"
   private val KillManifest = "K"
   private val RemoteWatcherHBManifest = "RWHB"
+  private val DoneManifest = "DONE"
```

Comment:
or just "D"?

Reply:
`"D"` is taken already. :-( `private val StatusSuccessManifest = "D"`

Reply:
But I agree that a shorter manifest could be used. `AD` as in `Akka Done` or maybe `DN`?

Reply:
yes, shorter saves a few bytes but doesn't matter much.
Personally, I don't try to be clever about the naming of those and prefer unrelated letters, "AA", "AB", "AC", "BA", "BB", "CA".

Here there is no convention, so "DN" sounds good.